Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -42,3 +43,44 @@
   - type: IdentityBlocker
     coverage: EYES
 
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons]
+  id: ClothingEyesPrescriptionSecurityGlasses
+  name: Prescription security glasses
+  description: A pair of security glasses with what appears to be a... prescription glass glued on top?.
+  components:
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Construction
+    graph: PrescriptionSecGlasses
```

Comment:
put the flashimmunity/eyeprotection/visioncorrection/identity blocker in an abstract prototype BasePrescriptionSecGlasses or something then have both of these inherit it for code deduplication